src/library/basesqltablemodel.cpp

```diff
@@ -723,6 +723,15 @@ QVariant BaseSqlTableModel::data(const QModelIndex& index, int role) const {
     // role
     switch (role) {
         case Qt::ToolTipRole:
+            if (column == fieldIndex(ColumnCache::COLUMN_LIBRARYTABLE_COLOR)) {
+                const auto color = mixxx::RgbColor::optional(value);
+                if (color) {
+                    value = mixxx::toQColor(color).name();
+                } else {
+                    value = QString();
+                }
+            }
+            [[fallthrough]];
```

Comment:
This causes unfortunately a warning. 
Please use M_FALLTHROUGH_INTENDED

Reply:
Fixed in #2518

Reply:
Thank you.